src/main/java/org/apache/pirk/responder/wideskies/ResponderDriver.java

```diff
@@ -49,83 +41,111 @@
 public class ResponderDriver
 {
   private static final Logger logger = LoggerFactory.getLogger(ResponderDriver.class);
+  // ClassNames to instantiate Platforms using the platform CLI
+  private final static String MAPREDUCE_LAUNCHER = "org.apache.pirk.responder.wideskies.mapreduce.MapReduceResponderLauncher";
+  private final static String SPARK_LAUNCHER = "org.apache.pirk.responder.wideskies.spark.SparkResponderLauncher";
+  private final static String SPARKSTREAMING_LAUNCHER = "org.apache.pirk.responder.wideskies.spark.streaming.SparkStreamingResponderLauncher";
+  private final static String STANDALONE_LAUNCHER = "org.apache.pirk.responder.wideskies.standalone.StandaloneResponderLauncher";
+  private final static String STORM_LAUNCHER = "org.apache.pirk.responder.wideskies.storm.StormResponderLauncher";
 
```

Comment:
Yes, I added this for backwards compatibility. Maybe overkill this early in the game, but didn't want to break anyone's scrips/bash history to quickly.